server/node/routes.js

```diff
@@ -15,7 +15,7 @@ const setup = require('./setup');
 const {products} = require('./inventory');
 const express = require('express');
 const router = express.Router();
-const stripe = require('stripe')(config.stripe.secretKey);
+const stripe = config.stripe.client;
```

Comment:
Use one single stripe client instance to make tracing calls to it throughout the app easy.